sql/src/ast.rs

```diff
@@ -107,6 +111,141 @@ pub struct ExistsTable {
     pub table_name: TableName,
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+pub struct CeresColumnOptionDef {
```

Comment:
The current implementation is a little bit complex and maybe a better and more simple way is to use `ColumnOption::DialectSpecific` provided by `sql-parser`.

And I guess you can refer to the implementation for `comment`:
https://github.com/CeresDB/ceresdb/blob/037735eb80a27f2cd476a4e1d443747fbee098fc/sql/src/parser.rs#L474

Reply:
We can record the `TimestampKey` information as `ColumnOption::DialectSpecific` in `ColumnDef.options` and afterwards we read that during generating `ColumnSchema`:
https://github.com/CeresDB/ceresdb/blob/037735eb80a27f2cd476a4e1d443747fbee098fc/sql/src/planner.rs#L668

Reply:
ok🧐